readme.md

```diff
@@ -1088,14 +1088,14 @@ namespace Azure.Service.Models
 
 ```yaml
 # autorest-core version
-version: 3.7.6
+version: 3.8.4
+modelerfour:
+  lenient-model-deduplication: true
```

Comment:
Please do not put this flag in a global places like here. If it is needed, please add it to your individual readme files of the test projects.

Reply:
Fixed.